src/layers/conv.jl

```diff
@@ -501,6 +504,27 @@ function _conv_size_check(layer, x::AbstractArray)
     lazy" expects size(input, $d) == $n, but got ", summary(x))))
 end
 ChainRulesCore.@non_differentiable _conv_size_check(::Any, ::Any)
+
+"""
+    _sizecheck(f, sz::Integer...)
+
+Ensures that the output of `f(sz...)` has the expected shape `sz`.
+
+Constructs a tensor using the function `f` with the given size `sz` and verifies that its shape matches `sz`.  
+If the shape does not match, a `DimensionMismatch` error is thrown.
+
+This is internally used to validate weight initialization functions.
+"""
+
+function _sizecheck(f, sz::Integer...)
```

Comment:
Maybe no space here?
```suggestion
"""
function _sizecheck(f, sz::Integer...)
```